ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -6368,3 +6368,84 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
```

Comment:
Hrm, this should be on the master branch.  The variant should probably be called `__release-4.5` (or `__release-4-5`)

Reply:
> Hrm, this should be on the master branch.

Do you maybe mean that `--branch=release-4.5` must be changed or the entire block must be moved into another filer?

Reply:
Hi @smarterclayton , could you please clarify which changes are required? Thanks